### PR TITLE
Makes QM a head of staff, HoP is no longer head of supply.

### PR DESCRIPTION
**Pull request by Hardly3D**
## About The Pull Request

This PR turns the Quartermaster into a command role, replacing HoP as the new head of supply (They are still head of service and civilians, just not supply anymore).

With that change being made, that also means that QMs will no longer be eligible to roll for antags like all the other authority roles and they will also start with basic stuff such as a telescopic baton, PTSD and a command tablet as well as a flash inside their locker like all the other heads for self-defense.

Mapping changes were also done to all maps (in rotation) to add a head console in QM's office, facilitating the process of promoting and demoting people to the supply department, as well as a keycard authentication.

Some minor changes were also done:
-Replaced the QM's headset found in Cargodrobe's contraband with two Cargonia banners (which don't heal in case you're worried)
-Cargo techies and miners no longer have access to QM's office during skeleton access

Minor changes unrelated to this PR were also made, such as removing Brig Phys from the blacklist, allowing brig physes slots to be raised considering that they became eligible to become antags not too long ago.

## Why It's Good For The Game

Makes more sense to give QM more power to the department they boss around, considering that most of the time their supervisor, HoP, will at best only give Cargo one or two visits to see how they are doing and will rarely give Cargo any orders, QM on the other hand is always on their own department.

And Cargo is also a newbie friendly department so it'll also allow newbies to trust QMs more, instead of risking being backstabbed by them, as well as making QM be held to higher standards like the other heads.

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

Latejoin window
image

Joining as the only head
image

New quartermaster office
image

Department console
image

**MAP CHANGES**

MetaStation

Old (from the webmap)
image

New
image

DeltaStation

Old
image

New
image

FlandStation

Old
image

New
image

PubbyStation

Old
image

New
image

KiloStation

Old
image

New
image

</details>

## Changelog
:cl:Hardly
tweak: The Quartermaster has finally received their promotion to a head of staff and thus, are now a head of supply!
tweak: The Head of Personnel in the other hand, is no longer the head of supply.
tweak: The QM's headset in Cargodrobe contraband has been replaced with two cargonia banners.
tweak: HoPs can now increase slots for Brig Physicians
/:cl:

### CODE REVIEW

**Comment by Coding-Grape** [comment]
Would they also get an extendable batong

**Comment by AnCopper** [comment]
One of the reason why the HoP is the head of two departments is because service alone has little tasks for a Head of Staff. There isn't enough to do.

**Comment by Mat05usz** [comment]
> One of the reason why the HoP is the head of two departments is because service alone has little tasks for a Head of Staff. There isn't enough to do.

Supply also has little tasks for the HoP because QM takes care of everything. Imo it's a good change, HoP should be a captain's right hand man/woman with a very minor responsibility being service

**Comment by EvilDragonfiend** [comment]
> One of the reason why the HoP is the head of two departments is because service alone has little tasks for a Head of Staff. There isn't enough to do.

HoP is likely considered to be a fancy version of assistant for captain. They shouldn't take a lot of works to learn how to be a captain from assisting their captain at their near side, but HoP is always busy for Cargo or something else. It's been my problem to find out how to be a captain when I had been a newbie HoP.

**Comment by Cartlord** [comment]
Maybe this will free up enough time for the Head of Personnel to check up on Service every few days.

**Comment by kit-katz** [comment]
cant wait to see how the qm's will now act to revs

**Comment by williamcll** [comment]
> One of the reason why the HoP is the head of two departments is because service alone has little tasks for a Head of Staff. There isn't enough to do.

The HoP Really should be monitoring the clown, mime and botany more.

**Comment by mystery3525** [comment]
I am not in support or opposition of this PR (except for the name, Head of Economics/Economy is better), but I remember reading a [forums post](https://forums.beestation13.com/t/qm-should-be-a-command-role-and-other-changes-to-the-command-structure/13281?u=mystery3525) detailing this exact same issue, major points below:
- QM being Head of Staff means that revolutionaries cannot gain as much headway in the Supply department as before, since the QM will have the knowledge of revolutionaries and will call it out on Command. More importantly, the budget card he holds is key, an easy withdraw round-start funds to prevent waste and secure all other budgets, storm cargo and order mindshields.
- QM can no longer be antagonist, how would non-murderbone targeting rules apply to them as a head? There isn't really any "steal" objectives for Cargo whereas all other department have at least two and there are rarely any roundstart CT's (of course if antag gets QM target then guh).
- Where would QM fall under the line of succession for Acting Captainship? (HoP > RD > CE > CMO > HoS) (**You need to add this**) (`var/list/chain_of_command` inside of the Jobs Subsystem)

**Review by PowerfulBacon — DISMISSED** [review]
Weird, but ok

**Comment by mcmeiler** [comment]
Can't wait to command ban people for being shitty QMs. You can't trust current QM players as command staff. They use the role as cargo tech+ , which is already assistant+ .

**Comment by Chirynnn** [comment]
Just merge it already.

**Comment by williamcll** [comment]
> Can't wait to command ban people for being shitty QMs. You can't trust current QM players as command staff. They use the role as cargo tech+ , which is already assistant+ .

That's just the same as people abusing Hop by giving themselves AA roundstart even when there's a captain.

**Comment by AnCopper** [comment]
> > Can't wait to command ban people for being shitty QMs. You can't trust current QM players as command staff. They use the role as cargo tech+ , which is already assistant+ .
> 
> That's just the same as people abusing Hop by giving themselves AA roundstart even when there's a captain.

That's;
1. Grounds for demotion.
2. Powergaming.

File: code/controllers/subsystem/job.dm
Comment (Archanial) [inline code/controllers/subsystem/job.dm]: Hos should always be the last in the list.
Plus this requires wiki changes.

Comment (Hardly3D) [reply code/controllers/subsystem/job.dm]: I have changed that with https://github.com/BeeStation/BeeStation-Hornet/pull/6833/commits/60363fa432fd71ffea7e1b6a718122204c1a4711, for wiki changes I've made a sandbox page for what to change.

**Comment by Hardly3D** [comment]
Guh, I thought the merge conflict was fixed with the first merge PR, guess not.

**Comment by PowerfulBacon** [comment]
Honestly, still not a huge fan of the idea. I did like the idea of something like a head of commerce or a proper head being made rather than the quartermaster being promoted to a head

**Comment by PowerfulBacon** [comment]
Not a fan of this PR, but I'm not actively against it enough to not let it in. It needs a second opinion.

**Comment by NormalAnomaly** [comment]
All of this looks like its a justification for the enduring condition of HoPs to take the one shred of non-ID Management job they have and ignore it.  Cargo play works in part because the head ISN'T living in the department, and people play QM because it ISN'T a head.  What is sketchy but allowed to slide for the QM to do currently becomes a serious SoP/Job Ban bait if he is made a head.  As long as the QM doesn't cause the station to blow up, nobody really cares when he cracks crates open to make a tesla bounty or any number of other things he does as part of the normal course of his job that he technically should not do but practicality (and low population at times) demands he do.  

The HoP not delivering orders generally speaking is not a problem, the job of cargo is mostly just handling the occasional crate, mining, and bounties.  The only time it makes much sense for the HoP to step in is when cargo needs to do something other than those tasks which is not often at all.  And with that in mind, most RDs I've worked under never issue orders, most CEs only say "Somebody make the SM." or they wordlessly do it themselves, CMO usually is just a really confident MD with viro access and chem access.  HoP doesn't just not order cargo, he also doesn't order service either.  How often does HoP step into botany and order them to make food for the chef, or demand that the chaplain hold a mass, or that the curator stock the shelves?

Actual newbies fresh to the game generally won't know they can't trust their QM, or else they won't in particular know that they CAN trust him if he is a head.

The QM being potentially corrupted is one of the cornerstones of revs.  Both command and the revs want to rush to secure the QM to order mindshields or to slow/prevent them getting ordered.  The QM is an important wildcard.

**Comment by EvilDragonfiend** [comment]
> Honestly, still not a huge fan of the idea. I did like the idea of something like a head of commerce or a proper head being made rather than the quartermaster being promoted to a head

I am making a PR to refactor the bank system. but I don't want to allow HoP to handle all finance things. What I am thinking is
* HoP can adjust all people's access, but can't adjust payment except for their department
* QM can adjust all people's payment regardless of their department, but can only adjust cargo access.

**Comment by EvilDragonfiend** [comment]
https://github.com/BeeStation/BeeStation-Hornet/pull/7559

So, this is what I made in the bank system. I don't want HoP handle this as their own, and I want give this power to QM, so that they can become head of finance as Bacon said.

**Comment by Hardly3D** [comment]
Let me make maintainers' job a little easier and just close this myself, I forgot about this PR and it's got a lot of merge conflicts that I can't be bothered to fix right now, along with how I'm currently taking a break from Bee. I'm looking forward to see EvilDragonFiend's idea go through though.

**Pull request closed without merging**
